index.js

```diff
@@ -88,20 +88,24 @@ NodePreGypGithub.prototype.createRelease = function(args, callback) {
 	});
 	
 	this.github.authenticate(this.authenticate_settings());
-	this.github.releases.createRelease(options, callback);
+	this.github.repos.createRelease(options, callback);
 };
 
 NodePreGypGithub.prototype.uploadAsset = function(cfg){
 	this.github.authenticate(this.authenticate_settings());
-	this.github.releases.uploadAsset({
+	if (this.release.id === undefined && this.release.data === undefined) {
```

Comment:
Why is this necessary? Where does release.data come from? Why now default the release id?

Reply:
In the new version the data that I receive from response it come into an object .data(in case there are multiple information to give) or just the single object. The release.id is necessary if the member is not defined you give a default value. If you have a suggestion I be glad to look into it and change it.